projects/cds/src/cms-components/merchandising/carousel/merchandising-carousel/merchandising-carousel.component.ts

```diff
@@ -0,0 +1,48 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { Product } from '@spartacus/core';
+import { Observable, of } from 'rxjs';
+import { filter, flatMap, map } from 'rxjs/operators';
+import { CmsComponentData } from '../../../../../../storefrontlib/src/cms-structure/page/model/cms-component-data';
```

Comment:
Please use `@spartacus/storefront` instead

Reply:
Removed this file for this PR, will be added for the cms component PR